models/issue.go

```diff
@@ -1388,6 +1389,9 @@ func GetUserIssueStats(opts UserIssueStatsOptions) (*IssueStats, error) {
 	if opts.RepoID > 0 {
 		cond = cond.And(builder.Eq{"issue.repo_id": opts.RepoID})
```

Comment:
Instead of adding conditional to SQL, could this just append `opts.RepoID` to `opts.RepoIDs` slice?

Reply:
I checked where UserIssueStatsOptions is used ... we can drop RepoID .... since it is only used in:
 * ./routers/user/home.go: 1time
* ./models/issue.go: 1time
* ./models/issue_test.go: 1time

Reply:
done ...